best-practices/javascript_debugging.md

```diff
@@ -0,0 +1,8 @@
+# Debugging in Javascript
+
+If you'd like to use stepwise debugging in javascript, consider using the following procedure when working in VSCode.
+
+* Type <CMD><Shift>p to bring up the command palette
+* Search for “Debug: JavaScript Debug Terminal”
+* In the terminal enter your test invocation, e.g. npx jest __test__/components/Whatever.test.js
```

Comment:
```suggestion
* In the terminal enter your test invocation, e.g. `npx jest __test__/components/Whatever.test.js`
```

and then MD wants to render `__test__` as bold if there are no formatting backticks.